microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -596,6 +596,26 @@ Microsoft also sends the product price, product image, product name, ratings, an
 
 The Microsoft Edge shopping feature requires sharing of cookie information with Bing.  For example, the cookies may be used for debugging, fraud detection, and analytics.  The Microsoft shopping service returns prices from other retailers, historical price trends, and any available coupons for that website.
 
+![Shopping and coupons](./media/inboxcoupons.png)
+
+To help you find and keep track of coupons in your inbox, Microsoft Edge may optionally scan your connected email account for coupons in promotional emails. The Microsoft shopping service extracts coupons from promotional emails sent by supported retailers. The extracted coupons are stored and managed by the Microsoft shopping service and is accessible to you. Inbox coupons will not appear when using **Guest mode** or **InPrivate**.
+
+![Shopping and coupons](./media/inboxcoupons-settings.png)
+
+To control which retailers' emails get scanned by Microsoft Edge, complete the following actions: 
+
+1.  Click on the **“Your Shopping”** tab on the shopping sidebar 
```

Comment:
missing step 1:  how do I get to the shopping sidebar?  I'm checking actual UI now...